pypolychord/output.py

```diff
@@ -4,8 +4,14 @@
     pass
 import re
 import os
+import numpy as np
+import pandas as pd
+import collections
 
 
+# currently breaking install
```

Comment:
Probably best to remove these comment lines if it's no longer included, along with the commented out display lines earlier

Reply:
I was thinking to leave them to remind me that at some point we should fix the ipython deps. I do not really understand why ipython is failing in the test.

Reply:
It looks like IPython >=7.0 only supports python 3.5 and above, so the tests fail on 2.7 and 3.4. I could be persuaded to stop supporting 3.4, but stopping 2.7 is a bit more of an ask. I reckon the best way to fix this is to add conditionals to the requirements.txt file, for example:

```
ipython < 6; python_version < '3.5'
```